### PR TITLE
CIS test for sa login disabled and login named sa does not exist

### PR DESCRIPTION
**Pull request by tboggiano**
# A New PR

THANK YOU - We love to get PR's and really appreciate your time and help to improve this module

## Accepting a PR

Before we accept the PR - please confirm that you have run the tests locally to avoid our automated build and release process failing. You can see how to do that in our wiki

https://github.com/sqlcollaborative/dbachecks/wiki 

## Please confirm you have 0 failing Pester Tests

[X] There are 0 failing Pester tests

## Changes this PR brings

- CIS test for if original sa login as been disabled
- CIS test to verify not login exist named sa

Please add below the changes that this PR covers. It doesnt need an essay just the bullet points :-)

### CODE REVIEW

**Comment by tboggiano** [comment]
A new trick for you to teach me Rob how to pull down changes before I put in a pull request so I can resolve merge conflicts before submitting the pull request.

**Comment by SQLDBAWithABeard** [comment]
To sync the upstream repository 

````
Git status
git fetch upstream
Git checkout branch that you are working on
git merge upstream/master
````

**Comment by tboggiano** [comment]
Yay! I'm getting good at this.

File: internal/configurations/configuration.ps1 [hunk 1]
```diff
@@ -58,6 +58,8 @@ Set-PSFConfig -Module dbachecks -Name policy.security.oleautomationproceduresdis
 Set-PSFConfig -Module dbachecks -Name policy.security.remoteaccessdisabled -Value 0 -Initialize -Description "Remote Access should be disabled 0"
 Set-PSFConfig -Module dbachecks -Name policy.security.scanforstartupproceduresdisabled -Validation bool -Value $true -Initialize -Description "Scan For Startup Procedures disabled `$true or enabled `$false"
 Set-PSFConfig -Module dbachecks -Name policy.security.latestbuild -Validation bool -Value $true -Initialize -Description "SQL Server should have the latest SQL build (service packs/CUs) installed"
+Set-PSFConfig -Module dbachecks -Name policy.security.sadisabled -Validation bool -Value $true -Initialize -Description "The sa login should be disabled"
+Set-PSFConfig -Module dbachecks -Name policy.security.saexist -Validation bool -Value $false -Initialize -Description "A login named sa does not exist"
```
[the remark this answers is not in the document]
Comment (SQLDBAWithABeard) [reply internal/configurations/configuration.ps1]: @tboggiano - Please can we have a -skip config for  these two checks - Sorry, I should have thought of that with my first review :-(

[the remark this answers is not in the document]
Comment (tboggiano) [reply internal/configurations/configuration.ps1]: Yay again I'm getting good at this I think!

**Pull request merged**